apps/test/unit/templates/sectionProgress/progressTables/ProgressTableSummaryCellTest.jsx

```diff
@@ -12,22 +12,36 @@ const DEFAULT_PROPS = {
     isStarted: true,
     completedPercent: 100,
     imperfectPercent: 0,
-    incompletePercent: 0
+    incompletePercent: 0,
+    timeSpent: 0,
+    lastTimestamp: 0
   },
   isAssessmentLesson: false,
   onSelectDetailView: () => {}
 };
 
 const setUp = (overrideProps = {}) => {
-  const props = _.merge(DEFAULT_PROPS, overrideProps);
-  return shallow(<ProgressTableSummaryCell {...props} />);
+  const props = _.merge(_.cloneDeep(DEFAULT_PROPS), overrideProps);
+  return (
+    mount(<ProgressTableSummaryCell {...props} />)
+      // <BorderedBox>
```

Comment:
I would expect `BorderedBox` to be tested through unit tests independently. And then Check that ProgressTableSummaryCell renders the BorderedBox with expected props. But that's a bit of a nit pick, so you can decide to ignore this comment if you want

Reply:
yeah i debated whether or not to do that. since that's your instinct i'll go ahead and update.